### DISCUSSION TITLE
Autosart services after reboot

### DISCUSSION THREAD

**Discussion by gdenas** [hsakoh/switchbot-mqtt#38] [Q&A] [answered]
Every time I reboot HA (eg updates) all services of SwitchBot-MQTT stop and have to be started manually. 
is there a way to autostart the services?

**Comment by hsakoh** [comment, marked as answer]
There are configuration items in the add-on settings.
`AutoStartServices`

https://github.com/hsakoh/switchbot-mqtt/wiki/HowToInstall

**Reply by gdenas** [reply]
Excellent. Done thank you
